diesel_cli/src/database.rs

```diff
@@ -316,8 +316,16 @@ pub fn database_url(matches: &ArgMatches) -> String {
 #[cfg(any(feature="postgres", feature="mysql"))]
 fn change_database_of_url(database_url: &str, default_database: &str) -> (String, String) {
     let mut split: Vec<&str> = database_url.split('/').collect();
-    let database = split.pop().unwrap();
-    let new_url = format!("{}/{}", split.join("/"), default_database);
+    let database_name_with_arguments: Vec<&str> = split.pop().unwrap().split('?').collect();
```

Comment:
Can we use the URL crate for this rather than doing our own parsing?

Reply:
@sgrif You mean code looking like this PR? https://github.com/diesel-rs/diesel/pull/861/files